### ISSUE TITLE
[Bug] Very slow page loading when using cloud drives

### ISSUE THREAD

**Issue by thedukegr** [gotson/komga#204]
### Komga environment
- **OS**: Ubuntu 18.04
- **Komga version**: Latest
- [x] **I am running Komga with Docker**
  - Docker image tag [e.g. latest, beta]: latest
- [ ] **I am running Komga from the `jar`**
  - Java version: 
- [x] **I have a problem in the web interface**
  - Browser (with version): Chrome 83.0.4103.97
- [ ] **I have a problem with an OPDS client application**
  - OPDS Application (with version): 
- [ ] **I have a problem with the Tachiyomi extension**
  - Tachiyomi version: 
  - Tachiyomi extension version: 

### Describe the bug
I have a big comic/manga library which is hosted in google drive, i have the cloud drive mounted as a local drive and its picked up by komga properly. It loads and analyzes everything fine. 
The problem is that when trying to read a book it takes too long to load the next page, sometimes upto 10 seconds, with large books this is even more apparent, lastly the thumbnail page viewer almost never loads.

### Expected behavior
I would expect pages to load almost instantly considering i got a fiber connection
### Actual behavior
It takes ages to load anything
### Additional context
I assume this is some problem with using google drive to host the library, what i dont understand is that for example plex can stream a 4k 80gb movie in like 2 seconds, how isit possible it takes like 10 seconds to  load one page of a 50mb comic? Im sure there can be some way to optimize the loading from cloud drives or even create some kind of cache.
### Log file
If applicable, add an excerpt of the log file (max 20 lines), or a link to a gist containing the log file.

**Comment by gotson** [comment]
Komga will open the archives and then fetch files individually to serve them.

It is not designed to work with remote drives. 

How is your Google drive setup exactly?

**Comment by gotson** [comment]
Closing this as there is no feedback. Please reopen if you need with the information requested (ie do you use Plexdrive or rclone, and provide configuration).

**Comment by snohotest** [comment]
Hi, sorry for bumping an old thread, but I was super confused why Komga seemed to load things super slowly for me until I found this issue via Google. 

I store most of my comics long-term in Google Drive and I was using rclone to mount that. Like the previous user it was taking about 10-30 seconds to load a comic initially and the application would buffer every three pages or so if I clicked next too quickly.

I'm not sure what else I can provide to help out here, but I this is the command I'm using to mount the drive

```
rclone mount gdrive:Comics/ ./mount
```

I was played around a little bit to see why this was happening but to no luck (upping openjdk version, giving more RAM), but once I read this thread, I manually copied a few files over onto the local drive and Komga began loading instantly!

As a workaround, I'm just going to copy the files onto the local drive as I need them, but it would be nice to have support for mounted cloud drives if possible. I just happen to be cheap and enjoy hosting my media server on free tier level cloud products 😄 with low local disk space and mounted drives.
